GVRf/Framework/framework/src/main/java/org/gearvrf/GVRViewManager.java

```diff
@@ -197,11 +197,13 @@ void onSurfaceCreated() {
         }
 
         mPreviousTimeNanos = GVRTime.getCurrentTime();
+
+
+        final GVRScene scene = null == mMainScene ? new GVRScene(GVRViewManager.this) : mMainScene;
+        setMainSceneImpl(scene);
```

Comment:
Why did this section had to move up?

Reply:
because makerendleBundle() has dependency for the mainscene. We create renderTarget in there which needs scene.